databases.go

```diff
@@ -277,6 +277,29 @@ func (c *Client) DatabaseCreateUser(ctx context.Context, app, addonID string, us
 	return res.DatabaseUser, nil
 }
 
+type DatabaseUpdateUserParam struct {
+	DatabaseID           string `json:"database_id"`
+	Password             string `json:"password,omitempty"`
+	PasswordConfirmation string `json:"password_confirmation,omitempty"`
+}
+
+type databaseUpdateUserPayload struct {
+	DatabaseUser DatabaseUpdateUserParam `json:"database_user"`
+}
+
+// DatabaseUpdateUser updates an user to the given database addon
+func (c *Client) DatabaseUpdateUser(ctx context.Context, app, addonID, username string, user DatabaseUpdateUserParam) (DatabaseUser, error) {
```

Comment:
suggestion: the "user" variable is misleading in this case. It could be better to name it like "databaseUserParams" WDYT?